### PR TITLE
Fix error in encoding exception in Chinese environment

### PR DESCRIPTION
**Pull request by mk8310**

### CODE REVIEW

**Comment by mk8310** [comment]
@dy93 ,Hi!
I don't understand about this line code:
```javascript
const toDeflate = unescape(encodeURIComponent(block));
```
because I get unescape can't support most of the double byte char.
Waiting for your reply, thanks!

**Comment by dy93** [comment]
@mk8310 Thanks, I will take a look this weekend.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "gitbook-plugin-plantuml-cloud",
+  "name": "plantuml",
```
Comment (dy93) [inline package.json:2]: Why did you rename the package name?
As far as I know, gitbook-plugin-xxxx is the naming convention of gitbook plugin.

File: index.js [hunk 2]
```diff
@@ -12,81 +12,86 @@ require('shelljs/global');
...
-      const umlPath = output.resolve(options.umlPath);
-      mkdir('-p', umlPath);
-    },
-    'page:before': async function(page) {
-      let content = page.content;
-      const output = this.output;
+    hooks: {
+        'init': function () {
+            const output = this.output;
+            const config = this.config.values.pluginsConfig['plantuml'];
```
Comment (dy93) [inline index.js:59]: it seems to me that we should not change the package name
